airbyte-integrations/bases/base-java/build.gradle

```diff
@@ -5,6 +5,12 @@ plugins {
 
 dependencies {
     implementation 'commons-cli:commons-cli:1.4'
+    implementation group: 'org.apache.sshd', name: 'sshd-mina', version: '2.7.0'
```

Comment:
nit: formatting doesn't match the `'commons-cli:commons-cli:1.4'` style we're using elsewhere.